apps/user/src/components/main/ApplicationSubmittionBox/ApplicationPeriodBox/ApplicationPeriodBox.hooks.ts

```diff
@@ -0,0 +1,10 @@
+import { 제출_마감_날짜, 제출_시작_날짜 } from '@/constants/form/constant';
+
+export const useSchoolRecruitDate = () => {
+  const submitStart = 제출_시작_날짜.format('YYYY년 MM월 DD일 (ddd) HH:mm');
+  const submitEnd = 제출_마감_날짜.format('YYYY년 MM월 DD일 (ddd) HH:mm');
```

Comment:
```suggestion
  const applicationStart = 제출_시작_날짜.format('YYYY년 MM월 DD일 (ddd) HH:mm');
  const applicationEnd = 제출_마감_날짜.format('YYYY년 MM월 DD일 (ddd) HH:mm');
```

네이밍 공통화를 위해서 다음과 같이 application으로 수정하는거에 대해서 어떻게 생각하시나용

Reply:
ApplicationPeriodBox 이니 application으로 바꾸는게 일관성 있겠네요👌